src/local/butler/reproduce.py

```diff
@@ -236,7 +235,8 @@ def _update_directory(directory_name, ignore_paths=None):
   environment.set_value('REPRODUCE_TOOL', 'True')
 
   # Force logging to console for this process and child processes.
-  environment.set_value('LOG_TO_CONSOLE', 'True')
+  if verbose:
+    environment.set_value('LOG_TO_CONSOLE', 'True')
```

Comment:
I'm not sure these log messages make sense to the average user. I tried to improve the output when running a bit to give the user faster feedback and set expectations that repeated runs will take a while.

Reply:
s/'True'/True in all places.

Reply:
Done!